.env.sample

```diff
@@ -17,6 +17,7 @@ ROCKET_ADDRESS=127.0.0.1
 # ROCKET_PORT=8000
 # ROCKET_ADDRESS=localhost
 WEBHOOK_TOKEN=some_random_token
+CORE_SERVICE_AUTH_TOKEN=some_other_random_token
```

Comment:
I'd change this to `TRANSACTION_SERVICE_AUTH_TOKEN` – I think this makes it more explicit for people onboarding to the infrastructure projects on what this is.